static/app/components/core/principles/motion/motion.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Motion
+layout: document
+description: Guidelines for using animation in Sentry's user interface.
+---
+
+import {MotionPlayground} from 'sentry/stories/playground/motion';
+
+Motion in the UI, when used thoughtfully, can bring interfaces to life and guide users through tasks. Effective animations provide feedback, reduce user frustration, and give a sense of direct manipulation of on-screen elements. They also make our product feel snappy and modern by conveying responsiveness. The following guidelines outline our motion design principles, best practices for using animation in common scenarios, and accessibility considerations.
+
+<MotionPlayground />
+
+# Principles
+
+## Purposeful
+
+Use animation only when it serves a clear purpose. Every motion should communicate something useful (e.g. feedback, a state change) and not exist just for decoration. If an animation doesn’t help the user achieve a goal or learn something, reconsider using it.
+
+## Responsive
+
+Keep animations fast and efficient so the interface feels snappy. Most UI animations should complete in under 600ms. Very short durations (around 100ms) are perceived as instantaneous—use these for simple micro-interactions so users never feel slowed down. Strike a balance: slow enough to be noticed and understood, but never so slow that the user is waiting on the animation.
+
+## Natural
+
+Animations should feel intuitive and natural, often by mimicking real-world behavior. Objects on screen should accelerate and decelerate smoothly (nothing jerky or purely linear) to follow a sense of light-weight physical physics. Incorporate techniques like easing (speeding up then slowing down) and arcs in movement paths to emulate how things move in life. The goal is for animations to be so fitting that they almost go unnoticed.
+
+## Accessible
+
+Design animations to include everyone. No essential information should ever be conveyed by motion alone. Ensure there are accompanying cues (like text or icons). Always consider users who experience motion sensitivity or other disabilities—we should provide a way to reduce or disable motion and avoid effects that can cause discomfort (details in the Accessibility section below).
+
+# Usage
+
+## Productive Motion
+
+Motion is a useful tool for emulating _tactility_, which can enhance interactions within components. _Productive_ motion is used as a result of direct user interaction, such as pressing a button. These are typically subtle micro-interactions that bring the interface to life and should be fast. Wherever possible, refer to satisfying physical motion as inspiration.
```

Comment:
It still isn't clear to me what productive motion means. Is it necessary to introduce this term? The points made in the descriptions seem to be covered in the principles above already.